infra/recipes/terraform/azure/versions.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+    required_providers {
+        azurerm = {
+            source = "hashicorp/azurerm"
+            version = "< 3.90.0" 
```

Comment:
Why this version is less than 3.90.0?

Reply:
It was giving me the error described in [this](https://github.com/hashicorp/terraform-provider-azurerm/issues/24804) thread. Let me know if I should find another workaround though.